news/_posts/core-weekly/2020-03-31-coreweekly-week-13.md

```diff
@@ -0,0 +1,194 @@
+* [#18272](https://github.com/PrestaShop/PrestaShop/pull/18272): Functional tests - Add test 'Export Orders', by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#18269](https://github.com/PrestaShop/PrestaShop/pull/18269): Functional tests - Add test 'Help card' for Stocks page. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#18268](https://github.com/PrestaShop/PrestaShop/pull/18268): Functional tests - Add test 'Help card' for Monitoring page. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#18267](https://github.com/PrestaShop/PrestaShop/pull/18267): Functional tests - Fix update successful message on addresses page, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#18250](https://github.com/PrestaShop/PrestaShop/pull/18250): Functional tests - Add test 'Filter orders', by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#18242](https://github.com/PrestaShop/PrestaShop/pull/18242): Functional tests - Add test 'Help card' for orders page, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#18231](https://github.com/PrestaShop/PrestaShop/pull/18231): Functional tests - Fix default timeout for waiting for selector, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#18197](https://github.com/PrestaShop/PrestaShop/pull/18197): Functional tests - Order settings enable/disable reordering option. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+* [#18161](https://github.com/PrestaShop/PrestaShop/pull/18161): Functional tests - Add test Enable and disable B2B mode, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#17965](https://github.com/PrestaShop/PrestaShop/pull/17965): Add HTML IDs and classes to View Customer BO page, by [@matks](https://github.com/matks)
+
+
+## Code changes in the '1.7.6.x' branch
+
+
+### Back office
+* [#18064](https://github.com/PrestaShop/PrestaShop/pull/18064): Fix error when trying to translate Serbian using the BO interface, by [@eternoendless](https://github.com/eternoendless)
+* [#17920](https://github.com/PrestaShop/PrestaShop/pull/17920): Wrong redirection when using the quick search for a category. Thank you [@PululuK](https://github.com/PululuK)
+
+
+### Front office
+* [#18103](https://github.com/PrestaShop/PrestaShop/pull/18103): Duplicate address when submitting a form with errors, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Tests
+* [#18309](https://github.com/PrestaShop/PrestaShop/pull/18309): Change test fixtures that need to be in the future, by [@jolelievre](https://github.com/jolelievre)
+
+
+## Code changes in modules, themes & tools
+
+
+### Changes in developer documentation
+* [#495](https://github.com/PrestaShop/docs/pull/495): Change node version as we no support 12.x LTS, by [@NeOMakinG](https://github.com/NeOMakinG)
```

Comment:
```suggestion
* [#495](https://github.com/PrestaShop/docs/pull/495): Change node version as we now support 12.x LTS, by [@NeOMakinG](https://github.com/NeOMakinG)
```

Reply:
then could you please change also the title of the relative pull request.

Reply:
Done !

Reply:
thanks !